truss/env.go

```diff
@@ -0,0 +1,45 @@
+package truss
+
+import (
+	"bytes"
+	"fmt"
+)
+
+// EnvInput input
+type EnvInput struct {
+	Env         string
+	Kubeconfigs map[string]interface{}
+	KubeDir     string
+}
+
+// EnvironmentVars key/value pairs of environment variables that should be set in the shell
+type EnvironmentVars struct {
+	Kubeconfig string
+}
+
+// Env configures environment variables that should be set in the bash shell
+func Env(input *EnvInput) (EnvironmentVars, error) {
+	kubeconfigName := input.Kubeconfigs[input.Env]
+
+	if kubeconfigName == nil {
+		return EnvironmentVars{}, fmt.Errorf("No kubeconfig found for env %s", input.Env)
+	}
+
+	kubeconfig := fmt.Sprintf("%s%s", input.KubeDir, kubeconfigName)
+
+	environmentVars := EnvironmentVars{
+		Kubeconfig: kubeconfig,
+	}
+
+	return environmentVars, nil
+}
+
+// BashFormat formats environment variables for bash
+func (environmentVars *EnvironmentVars) BashFormat(env string) string {
```

Comment:
@jgramoll What do you think about this? I split these up into two different functions. The command is [doing a _bit_ more orchestration](https://github.com/instructure-bridge/truss-cli/pull/13/files#diff-debc89e3b3eacf90f19fb4e854a85d9eR35), but I do like this better.